docs/src/literate-tutorials/linear_elasticity.jl

```diff
@@ -1,3 +1,408 @@
-# # Linear elasticity
+# # [Linear elasticity](@id tutorial-linear-elasticity)
 #
-# TBW
+# ![](linear_elasticity.svg)
+#
+# *Figure 1*: Linear elastically deformed 1mm $\times$ 1mm Ferrite logo.
+#
+#md # !!! tip
+#md #     This tutorial is also available as a Jupyter notebook:
+#md #     [`linear_elasticity.ipynb`](@__NBVIEWER_ROOT_URL__/tutorials/linear_elasticity.ipynb).
+#-
+#
+# ## Introduction
+#
+# The classical first finite element problem to solve in solid mechanics is a linear balance
+# of momentum problem. We will use this to introduce a vector valued field, the displacements
+# $\boldsymbol{u}(\boldsymbol{x})$. In addition, some features of the
+# [`Tensors.jl`](https://github.com/Ferrite-FEM/Tensors.jl) toolbox are demonstrated.
+#
+# ### Strong form
+# The strong form of the balance of momentum for quasi-static loading is given by
+# ```math
+# \begin{alignat*}{2}
+#   \mathrm{div}(\boldsymbol{\sigma}) + \boldsymbol{b} &= 0 \quad &&\boldsymbol{x} \in \Omega \\
+#   \boldsymbol{u} &= \boldsymbol{u}_\mathrm{D} \quad &&\boldsymbol{x} \in \Gamma_\mathrm{D} \\
+#   \boldsymbol{n} \cdot \boldsymbol{\sigma} &= \boldsymbol{t}_\mathrm{N} \quad &&\boldsymbol{x} \in \Gamma_\mathrm{N}
+# \end{alignat*}
+# ```
+# where $\boldsymbol{\sigma}$ is the (Cauchy) stress tensor and $\boldsymbol{b}$ the body force.
+# The domain, $\Omega$, has the boundary $\Gamma$, consisting of a Dirichlet part, $\Gamma_\mathrm{D}$, and
+# a Neumann part, $\Gamma_\mathrm{N}$, with outward pointing normal vector $\boldsymbol{n}$.
+# $\boldsymbol{u}_\mathrm{D}$ denotes prescribed displacements on $\Gamma_\mathrm{D}$,
+# while $\boldsymbol{t}_\mathrm{N}$ the known tractions on $\Gamma_\mathrm{N}$.
+#
+# In this tutorial, we use linear elasticity, such that
+# ```math
+# \boldsymbol{\sigma} = \boldsymbol{\mathsf{C}} : \boldsymbol{\varepsilon}, \quad
```

Comment:
Maybe
```suggestion
# \boldsymbol{\sigma} = \boldsymbol{\mathbb{C}} : \boldsymbol{\varepsilon}, \quad
```
because at least on my machines the font is not really distinguishable from boldsymbol